src/App.tsx

```diff
@@ -1,10 +1,69 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useState, useEffect, useMemo } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoHeader } from './components/TodoHeader/TodoHeader';
+import { TodoList } from './components/TodoList/TodoList';
+import { TodoFilter } from './components/TodoFilter/TodoFilter';
+import { Notification } from './components/Notification/Notification';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { getTodos } from './api/todos';
+
+const USER_ID = 10140;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [error, setError] = useState<string | null>(null);
+
+  const [status, setStatus] = useState<string>('all');
+  const [itemsLeft, setItemsLeft] = useState<number>(0);
+  const [itemsCompleted, setitemsCompleted] = useState<number>(0);
```

Comment:
Typo in the state variable name, change 'setitemsCompleted' to 'setItemsCompleted'